docs/src/pages/components/dropdown.mdx

```diff
@@ -58,12 +59,13 @@ const Comp = () => (
 )
 export default Comp
 
-===
+```
 
----
```

Comment:
Oh, oops, you had this as a switcher.  

Somehow I may have missed that.  Hence my suggestion to remove the `description: ` prefix on one of these description lines.  

I'm sorry.  I am fine with the switcher, fwiw.